uaiohttpclient/example.py

```diff
@@ -16,8 +16,13 @@ def print_stream(resp):
 
 def run(url):
     resp = yield from aiohttp.request("GET", url)
-    print(resp)
-    yield from print_stream(resp)
+    try:
+        print(resp)
+        yield from print_stream(resp)
+    except:
+        pass
```

Comment:
This cannot be correct. And I always worry when I hear things like "This use case is absolutely necessary for production systems" and see code like this from the same person ;-). And wonder which part to trust, and I tend to trust the code. And code here, in a simple usecase, is incorrect. What can be said about more complex code? Please understand the logic of maintainers, who need to look thru bunch of code from many people. Simple things can't be incorrect, they put shade on the entire patchset.

Reply:
Why us that code incorrect. I have a good experience in Python (but very limited with async code), and I don't understand why this shouldn't work.

Maybe, are you upset with the pass statement inside except? I agree: it's ugly to send silently the exceptions to trash. But this is not library code, only a minimal example (here used as a documentation), and I just wanted to show that closing the stream was mandatory, not giving details om the exception handling which is application specific and out of the scope of the example.